rust/cymbal/src/langs/js.rs

```diff
@@ -81,43 +101,6 @@ impl RawJSFrame {
         Url::parse(&source_url[..useful])
             .map_err(|_| JsResolveErr::InvalidSourceUrl(source_url.to_string()))
     }
-
-    // JS frames can only handle JS resolution errors - errors at the network level
-    pub fn try_handle_resolution_error(&self, e: JsResolveErr) -> Result<Frame, Error> {
-        // A bit naughty, but for code like this, justified I think
-        use JsResolveErr::{
-            InvalidSourceMap, InvalidSourceMapHeader, InvalidSourceMapUrl, InvalidSourceUrl,
-            NoSourceUrl, NoSourcemap, TokenNotFound,
-        };
-
-        // I break out all the cases individually here, rather than using an _ to match all,
-        // because I want this to stop compiling if we add new cases
-        Ok(match e {
-            NoSourceUrl => self.try_assume_unminified().ok_or(NoSourceUrl), // We assume we're not minified
-            NoSourcemap(u) => self.try_assume_unminified().ok_or(NoSourcemap(u)),
-            InvalidSourceMap(e) => Err(JsResolveErr::from(e)),
-            TokenNotFound(s, l, c) => Err(TokenNotFound(s, l, c)),
-            InvalidSourceUrl(u) => Err(InvalidSourceUrl(u)),
-            InvalidSourceMapHeader(u) => Err(InvalidSourceMapHeader(u)),
-            InvalidSourceMapUrl(u) => Err(InvalidSourceMapUrl(u)),
-        }?)
-    }
-
-    // Returns none if the frame is
-    fn try_assume_unminified(&self) -> Option<Frame> {
```

Comment:
Will removing this mean that we have no way of knowing if something is unminified i.e. does not need demangling.  Not necessary right now as long as we know to come back to it

Reply:
I couldn't figure out a way to differentiate between unminified frames and simply errored ones - we should add this back once we figure out a good heuristic for that, but I think we can do that in time.